scripts/west_commands/create_board/templates/nrf53/board.cmake.jinja2

```diff
@@ -7,13 +7,14 @@ if(CONFIG_TFM_FLASH_MERGED_BINARY)
 endif()
 
 if(CONFIG_BOARD_{{ board | upper }}_NRF5340_CPUAPP OR CONFIG_BOARD_{{ board | upper }}_NRF5340_CPUAPP_NS)
+  board_runner_args(nrfutil "--ext-mem-config-file=${BOARD_DIR}/support/nrf5340dk_qspi_nrfutil_config.json")
```

Comment:
this too, a board might not have QSPI flash at all, or if it does it might have a very different configuration, on different pins, different mode, etc. plus I don't see this file even here?